targets/TARGET_STM/TARGET_STM32F2/gpio_irq_api.c

```diff
@@ -304,7 +304,7 @@ void gpio_irq_set(gpio_irq_t *obj, gpio_irq_event event, uint32_t enable)
                 mode = STM_MODE_IT_FALLING;
                 obj->event = EDGE_FALL;
             } else { // NONE or RISE
-                mode = STM_MODE_IT_EVT_RESET;
+                mode = STM_MODE_INPUT;
```

Comment:
Are these GPIO changes related to the assert changes? Or did these come in by mistake?